lib/fluent/plugin/out_rdkafka.rb

```diff
@@ -91,8 +92,22 @@ def configure(conf)
     super
     log.instance_eval {
       def add(level, &block)
-        if block
+        return unless block
+
+        # Follow rdkakfa's log level. See also rdkafka-ruby's bindings.rb.
```

Comment:
I think it's better to have the permanent link which has commit hash to bindings.rb.

Reply:
Added github link